src/native/NativeConstants.ts

```diff
@@ -176,6 +176,7 @@ interface NativeStringKey {
   reproStepsListEmptyStateDescription: any;
   reproStepsListHeader: any;
   reproStepsListItemTitle: any;
+  reproStepsListItemNumberingTitle: any;
```

Comment:
I have a question just for understanding, why did you give `reproStepsListItemNumberingTitle` the type `any`? Can it be anything else than `string`?

Reply:
Technically, we set it to a string in the native `ArgsRegistry`. So I think it's a valid point to change it to `string` (I'd say we also need to do the same for all the other keys, we should see what they map to and give them an appropriate type).
To not get out of the scope of this PR, I'll only change the type of this key for now, but let's look into changing all the other types as well.

Reply:
@asamiz Looks like TypeScript doesn't like computed string values in enums. Here's the error I get:
```
src/utils/Enums.ts:200:38 - error TS18033: Only numeric enums can have computed members, but this expression has type 'string'. If you do not need exhaustiveness checks, consider using an object literal instead.
```

Looking it up, looks like we either need to upgrade to TypeScript v5 or use an object instead of an enum.

I'll revert it back to `any` but let's look further into it and see what approach we will go with.